### PR TITLE
[chore] dependabot updates Tue Aug 22 19:28:14 UTC 2023

### PR DESCRIPTION
**Pull request by opentelemetrybot**
Bump cloud.google.com/go/spanner from 1.47.0 to 1.48.0 in /receiver/googlecloudspannerreceiver
Bump github.com/ClickHouse/clickhouse-go/v2 from 2.13.0 to 2.13.2 in /exporter/clickhouseexporter
Bump github.com/aliyun/aliyun-log-go-sdk from 0.1.51 to 0.1.53 in /exporter/alibabacloudlogserviceexporter
Bump github.com/antonmedv/expr from 1.13.0 to 1.14.2 in /pkg/stanza
Bump github.com/antonmedv/expr from 1.13.0 to 1.14.2 in /receiver/receivercreator
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.327 in /exporter/awscloudwatchlogsexporter
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.327 in /exporter/awsxrayexporter
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.327 in /extension/observer/ecsobserver
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.327 in /internal/aws/cwlogs
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.327 in /internal/aws/xray
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.327 in /internal/metadataproviders
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.327 in /processor/resourcedetectionprocessor
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.327 in /receiver/awscontainerinsightreceiver
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.327 in /receiver/awsecscontainermetricsreceiver
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.328 in /exporter/awsemfexporter
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.328 in /exporter/datadogexporter
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.328 in /exporter/kafkaexporter
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.328 in /internal/aws/awsutil
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.328 in /internal/aws/k8s
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.328 in /internal/aws/proxy
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.328 in /internal/aws/xray/testdata/sampleapp
Bump github.com/aws/aws-sdk-go from 1.44.323 to 1.44.328 in /receiver/awsxrayreceiver
Bump github.com/aws/aws-sdk-go-v2 from 1.20.1 to 1.20.3 in /extension/sigv4authextension
Bump github.com/aws/aws-sdk-go-v2 from 1.20.1 to 1.21.0 in /exporter/awskinesisexporter
Bump github.com/aws/aws-sdk-go-v2/config from 1.18.33 to 1.18.35 in /extension/sigv4authextension
Bump github.com/aws/aws-sdk-go-v2/config from 1.18.33 to 1.18.36 in /exporter/awskinesisexporter
Bump github.com/aws/aws-sdk-go-v2/credentials from 1.13.32 to 1.13.34 in /extension/sigv4authextension
Bump github.com/aws/aws-sdk-go-v2/credentials from 1.13.32 to 1.13.35 in /exporter/awskinesisexporter
Bump github.com/aws/aws-sdk-go-v2/service/kinesis from 1.18.2 to 1.18.5 in /exporter/awskinesisexporter
Bump github.com/aws/aws-sdk-go-v2/service/sts from 1.21.2 to 1.21.4 in /extension/sigv4authextension
Bump github.com/google/uuid from 1.3.0 to 1.3.1 in /cmd/configschema
Bump github.com/google/uuid from 1.3.0 to 1.3.1 in /exporter/awscloudwatchlogsexporter
Bump github.com/google/uuid from 1.3.0 to 1.3.1 in /exporter/awsemfexporter
Bump github.com/google/uuid from 1.3.0 to 1.3.1 in /exporter/googlecloudpubsubexporter
Bump github.com/google/uuid from 1.3.0 to 1.3.1 in /exporter/tanzuobservabilityexporter
Bump github.com/google/uuid from 1.3.0 to 1.3.1 in /pkg/ottl
Bump github.com/google/uuid from 1.3.0 to 1.3.1 in /processor/k8sattributesprocessor
Bump github.com/google/uuid from 1.3.0 to 1.3.1 in /processor/tailsamplingprocessor
Bump github.com/google/uuid from 1.3.0 to 1.3.1 in /receiver/awsxrayreceiver
Bump github.com/google/uuid from 1.3.0 to 1.3.1 in /receiver/k8sclusterreceiver
Bump github.com/google/uuid from 1.3.0 to 1.3.1 in /receiver/skywalkingreceiver
Bump github.com/mongodb-forks/digest from 1.0.4 to 1.0.5 in /receiver/mongodbatlasreceiver
Bump github.com/sijms/go-ora/v2 from 2.7.11 to 2.7.14 in /receiver/sqlqueryreceiver
Bump github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common from 1.0.719 to 1.0.727 in /exporter/tencentcloudlogserviceexporter
Bump go.opentelemetry.io/collector/receiver from 0.82.0 to 0.83.0 in /receiver/googlecloudpubsubreceiver
Bump golang.org/x/vuln from 1.0.0 to 1.0.1 in /internal/tools
Bump gonum.org/v1/gonum from 0.13.0 to 0.14.0 in /pkg/stanza
Bump gonum.org/v1/gonum from 0.13.0 to 0.14.0 in /receiver/statsdreceiver
Bump google.golang.org/api from 0.136.0 to 0.138.0 in /exporter/f5cloudexporter
Bump google.golang.org/api from 0.136.0 to 0.138.0 in /exporter/googlecloudpubsubexporter
Bump google.golang.org/api from 0.136.0 to 0.138.0 in /receiver/googlecloudpubsubreceiver
Bump google.golang.org/api from 0.136.0 to 0.138.0 in /receiver/googlecloudspannerreceiver

### CODE REVIEW

File: cmd/configschema/go.mod [hunk 1]
```diff
@@ -4,8 +4,8 @@ go 1.20
 
 require (
 	github.com/fatih/structtag v1.2.0
-	github.com/google/uuid v1.3.0
-	github.com/open-telemetry/opentelemetry-collector-contrib v0.83.0
+	github.com/google/uuid v1.3.1
+	github.com/open-telemetry/opentelemetry-collector-contrib v0.0.0-00010101000000-000000000000
```
Comment (songy23) [inline cmd/configschema/go.mod:8]: Is this intended?

Comment (codeboten) [reply cmd/configschema/go.mod:8]: it is not, i had the same question here https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/26025#discussion_r1301816036

**Review by codeboten — COMMENTED** [review]
Still an error 😠

**Comment by codeboten** [comment]
Trying this one more time.... the `expr` dep update caused issues https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/26038

**Pull request closed without merging**